packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -20,7 +26,9 @@ spec:
           - name: ISTIO_NAMESPACE
             value: {{ template "istio.namespace" . }}
           - name: FORCE_INSTALL
-            value: {{ .Values.forceInstall | default "false" | quote }}
+            value: {{ .Values.installer.forceInstall | quote }}
```

Comment:
Why did we remove the default?

Reply:
Because its set in the values.yaml, whats point in keeping it both places ?

Reply:
this is added back in with the most recent commits - I originally removed it because we are setting this by default in the values.yaml so doesn't make sense to also have a default value - but it also doesn't hurt in this situation.